src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -391,6 +432,98 @@ def path_without_prefix(subvolume_path)
       def types_for_is
         super << :btrfs
       end
+
+    private
+
+      # Check for existing descendants of a subvolume path.
+      #
+      # @param path [String] subvolume path
+      #
+      # @return [Boolean]
+      #
+      def subvolume_descendants_exist?(path)
+        !subvolume_descendants(path).empty?
+      end
+
+      # Find the most suitable parent for a new subvolume
+      #
+      # That is, the one with the longest path that is part of path.
+      #
+      # @param path [String] subvolume path
+      #
+      # @return [BtrfsSubvolume] parent subvolume
+      #
+      def best_match(path)
```

Comment:
I'm the less suitable person to complain about names but `Btrfs#best_match` doesn't really look like the best name for a method to search a candidate parent.